openedx/features/wikimedia_features/meta_translations/transformers/wiki_transformer.py

```diff
@@ -165,3 +163,8 @@ def meta_data_to_raw_data(self, meta_data):
                 else:
                     raise Exception('{} not found in xml_data'.format(key))
         return etree.tostring(problem)
+
+TRANSFORMER_CLASS_MAPPING = {
```

Comment:
create settings folder in meta_translation app and move these settings there.
![image](https://user-images.githubusercontent.com/42185078/169233987-46d563cb-f687-4919-ab3f-924c2baec2ad.png)